w07h03/FieldVisualizeHelper.java

```diff
@@ -0,0 +1,117 @@
+package pgdp.sim;
```

Comment:
wrong file path then! imo this should be under `w07h03/test/pgdp/sim`

Reply:
hm or maybe not test but `helpers` or sth

Reply:
yeah i was looking at my DeGraphViz from W06H03, and there it was this location I think. But can change it if you like.

Reply:
yes please! @cod-eric